### PR TITLE
remove jquery and selectBox css

### PR DESCRIPTION
**Pull request by avimoondra**
Removes jquery and selectBox. I may have missed some items here, so pls have a 👀  thanks!

### CODE REVIEW

File: _sass/2-layout/_global.scss [hunk 1]
```diff
@@ -2,7 +2,6 @@
 @import "../1-modules/mixins";
 @import "../1-modules/colors";
 @import "../1-modules/header";
-@import "../1-modules/selectBox";
```
Comment (avimoondra) [inline _sass/2-layout/_global.scss:5]: the main change, everything else is formatting

File: package.json [hunk 2]
```diff
@@ -42,7 +42,6 @@
         "gatsby-transformer-remark": "^2.8.15",
         "gatsby-transformer-yaml": "^2.4.4",
         "jest": "25.5.4",
-        "jquery": "^3.5.1",
```
Comment (avimoondra) [inline package.json:45]: bye jquery 👋

File: src/components/common/Layout.tsx [hunk 3]
```diff
@@ -51,11 +51,6 @@ export default class Layout extends React.Component<LayoutProps> {
 
           {/* JS Includes */}
           {/* eslint-disable-next-line max-len*/}
-          <script
-            src="https://code.jquery.com/jquery-2.2.4.min.js"
-            integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44="
-            crossOrigin="anonymous"
-          ></script>
```
Comment (avimoondra) [inline src/components/common/Layout.tsx:58]: BYE jquery 👋

Comment (bingbongle) [reply src/components/common/Layout.tsx:58]: byebye

**Pull request merged**